app/models/page.rb

```diff
@@ -8,6 +8,13 @@ class Page < ApplicationRecord
   def to_param
     slug
   end
+
+  def self.promote(page)
+    transaction do
+      page.website.pages.update(landing: false)
```

Comment:
Can you inspect the db query here to ensure this isn't too bananas?

Reply:
@adarsh I am not sure if this helps: 
```
Page Load (0.2ms)  SELECT "pages".* FROM "pages" WHERE "pages"."website_id" = $1  [["website_id", 3]]
Page Update (0.2ms)  UPDATE "pages" SET "updated_at" = $1, "landing" = $2 WHERE "pages"."id" = $3  [["updated_at", "2022-04-15 15:37:39.961821"], ["landing", true], ["id", 4]]
```